rubocop.infer.yml

```diff
@@ -0,0 +1,16 @@
+<%
+  # Identify installed RuboCop plugins to configure
+  plugin_configs = {
+    "rubocop" => "rubocop.yml",
+    "rubocop-graphql" => "rubocop.graphql.yml",
+    "rubocop-minitest" => "rubocop.minitest.yml",
+    "rubocop-performance" => "rubocop.performance.yml",
+    "rubocop-rails" => "rubocop.rails.yml",
+    "rubocop-rake" => "rubocop.graphql.yml",
+    "rubocop-rspec" => "rubocop.rspec.yml",
+  }.select { |plugin_name, _| Gem.loaded_specs.include?(plugin_name) }
```

Comment:
The tricky part is that this checks if the gem is installed in the computer, not in the bundler right?

Reply:
Yes. The way it would work is that if a consumer has

```yml
# .rubocop.yml
inherit_gem:
  rubocop-shopify: rubocop.infer.yml
```

then when they run `rubocop` it will look at their installed gems, and inherit from the applicable configs.

Reply:
But if the app doesn't have rubocop-minitest for example but the person has it installed globally we should not use it.

We should be using what is on the Gemfile.lock of the application, not what is on the gem paths.

Reply:
Yes, good point. I should have said "in their Gemfile" not "installed". I just confirmed `Gem.loaded_specs` handles this the way we want:

```console
% cat Gemfile
source 'https://rubygems.org'
% bundle exec ruby -e 'puts Gem.loaded_specs.key?("rubocop-minitest")'
false
% gem install rubocop-minitest
Successfully installed rubocop-minitest-0.32.2
Parsing documentation for rubocop-minitest-0.32.2
Installing ri documentation for rubocop-minitest-0.32.2
Done installing documentation for rubocop-minitest after 0 seconds
1 gem installed
% bundle exec ruby -e 'puts Gem.loaded_specs.key?("rubocop-minitest")'
false
% bundle add rubocop-minitest > /dev/null && cat Gemfile
source 'https://rubygems.org'

gem "rubocop-minitest", "~> 0.32.2"
% bundle exec ruby -e 'puts Gem.loaded_specs.key?("rubocop-minitest")'
true
% bundle exec ruby -e 'puts Gem.loaded_specs.key?("rubocop")'
true
```